docs/source/datastructures_peak.rst

```diff
@@ -8,7 +8,7 @@ The most important container for raw data and peaks is ``MSSpectrum`` which we
 have already worked with in the `Getting Started <getting_started.html>`_
 tutorial.  ``MSSpectrum`` is a container for 1-dimensional peak data (a
 container of ``Peak1D``). You can access these objects directly, however it is
-faster to use the ``get_peaks()`` and ``set_peaks`` functions which use Python
+faster to use the ``get_peaks()`` and ``set_peaks()`` functions which use Python
 numpy arrays for raw data access. Meta-data is accessible through inheritance
 of the ``SpectrumSettings``  objects which handles meta data of a spectrum. 
```

Comment:
You might want to use crossreferences to the apidocs:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects

Reply:
https://github.com/OpenMS/pyopenms-docs/blob/master/docs/source/datastructures_peak.rst

Reply:
I tried to set up a readthedocs to test this out, but it doesn't take the most recent version, beside build triggering on every push. https://mp-pyopenms-docs.readthedocs.io/en/latest/datastructures_peak.html

Reply:
I tried writing `.. py:function:: get_peaks()`, but it didn't work.